exercises/microwave/.meta/solutions/microwave.rb

```diff
@@ -0,0 +1,14 @@
+class Microwave
+  attr_reader :minutes, :seconds
+
+  def initialize(input)
+    base = input > 99 ? 100 : 60
+
+    @minutes = input / base
+    @seconds = (input - minutes * base) % base
+  end
+
+  def timer
+    "#{'%02i' % minutes}:#{'%02i' % seconds}"
```

Comment:
Interpolating strings seems a bit redundant.

There is no restriction that format codes are the only thing in the string when formatting.

```suggestion
    `%02i:%02i' % [minutes, seconds]
```

Single quotes allowed as there is no interpolation happening.  We could even name them (which depending on style opinions, annotating is preferred).

```suggestion
    %02<minutes>i:%02<seconds>i' % {minutes: minutes, seconds: seconds}

Reply:
ccafd15